common/lib/xmodule/xmodule/modulestore/courseware_index.py

```diff
@@ -127,9 +130,40 @@ def remove_index_item_location(item_location):
         if raise_on_error and error_list:
             raise SearchIndexingError(_('Error(s) present during indexing'), error_list)
 
+        return indexed_count
+
+    @classmethod
+    def do_publish_index(cls, modulestore, location, delete=False, raise_on_error=False):
+        """
+        Add to courseware search index published section and children
+        """
+        indexed_count = cls.add_to_search_index(modulestore, location, delete, raise_on_error)
+        #Google Analytics - log index published section request
```

Comment:
It's great to get more analytics tracking log events here!  But I'm unclear what the tracking log calls below have to do with Google analytics.